src/cf-expandables/usage.md

```diff
@@ -78,27 +78,29 @@ Overwrite them in your own project by duplicating the variable `@key: value`.
 
 ### Expanded
 
-Sometimes you may want the expandable to be open by default.
-This is as easy as adding the `.o-expandable_content__onload-open` modifier to the `.o-expandable_content` block.
+Sometimes you may want the expandable to be open by default. This is as easy
+as adding the `.o-expandable_content__onload-open` modifier to the
+`.o-expandable_content` block.
 
 ```
 .o-expandable_content__onload-open
 ```
 
 ### Padded
 
-Adds padding and a background color to `.o-expandable_header` and `.o-expandable_content`.
+Adds `padding` and a `background` color to `.o-expandable_header` and
+`.o-expandable_content`.
 
-In addition to using the `.o-expandable__padded` modifier you also
-need to make sure you are using `.o-expandable_header`.
+In addition to using the `.o-expandable__padded` modifier you alsoneed to make
```

Comment:
What's semantic about the existing break here? If anything, there should be a comma after `modifier` and you could break it there, but having a break after `also` doesn't seem special to me.